advocacy_docs/partner_docs/CommvaultBackupandRecovery/02-PartnerInformation.mdx

```diff
@@ -1,12 +1,12 @@
 ---
 title: 'Partner information'
-description: 'Details for Commvault Backup & Recovery'
+description: 'Details of the Partner'
 
 ---
 | &nbsp;     | &nbsp; |
 | ----------- | ----------- |
 | **Partner Name**      | Commvault       |
 | **Partner product**   |  Commvault Backup & Recovery       |
 | **Website**      |    https://www.commvault.com/   |
-| **Version**   |  Commvault Backup & Recovery 11.24      |
+| **Version**   |  Commvault Backup & Recovery 11.32      |
 | **Product description**      |  Wherever your data resides, ensure availability by way of a single interface with Commvault Backup & Recovery. It offers domprehensive workload coverage, files, apps, and databases, including EDB Postgres Advanced Server and EDB Postgres Extended Server, from a single extensible platform and user interface. Commvault Backup & Recovery provides a comprehensive backup and archiving solution for your trusted recovery, ransomware protection, and security.     |
```

Comment:
Can you add Postgres to description?

including EDB Postgres Advanced Server, EDB Postgres Extended Server, and PostgreSQL, from a  ....